go.mod

```diff
@@ -441,11 +446,13 @@ replace (
 	github.com/vmware/govmomi => github.com/vmware/govmomi v0.20.3
 	github.com/willf/bitset => github.com/willf/bitset v1.1.11-0.20200630133818-d5bec3311243
 	github.com/xiang90/probing => github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2
+	github.com/xlab/treeprint => github.com/xlab/treeprint v0.0.0-20181112141820-a009c3971eca
 	github.com/yuin/goldmark => github.com/yuin/goldmark v1.2.1
 	go.etcd.io/bbolt => go.etcd.io/bbolt v1.3.5
 	go.etcd.io/etcd => go.etcd.io/etcd v0.5.0-alpha.5.0.20200910180754-dd1b699fc489 // ae9734ed278b is the SHA for git tag v3.4.13
 	go.mongodb.org/mongo-driver => go.mongodb.org/mongo-driver v1.1.2
 	go.opencensus.io => go.opencensus.io v0.22.3
+	go.starlark.net => go.starlark.net v0.0.0-20200306205701-8dd3e2ee1dd5
```

Comment:
this appears to add a parser/interpreter. some questions about the capabilities this exposes:
1. does the user have to do anything to opt into kustomize loading/evaluating code with this intepreter?
2. is code this interprets loaded from files on disk? if so, how are the paths specified, and can they be arbitrary or are they bounded to kustomize directories or other specific directories?
3. is code this interprets loaded over the network? if so, how are the source URLs specified?
4. what capabilities does the code this interprets have access to? can it load other files or make network calls? can it consume arbitrary CPU/memory?

Reply:
Starlark provides the possibility of a starlark-based plugin.  A user must install configuration for said plugin and add `-enable-star` to the `kubectl kustomize` command to allow the config to be honored.   It's marked alpha, we're not obligated to keep it.  I understand your concerns.  I'd prefer to remove the flag from kubectl than perform a security review in this PR.  Unfortunately, I cannot promise to remove the dep as well and make codefreeze.

Reply:
As long as it requires explicit opt-in from the user and is not enabled/exposed by default, I won't block on this. Long-term, the flag and dep should be partitioned so they can be imported/registered together, and not need to be part of the dependency tree if unused.